tests/jetty-ee10-test-maven-plugin/src/it/jetty-start-forked-mojo-it/pom.xml

```diff
@@ -38,4 +38,11 @@
     </dependencies>
   </dependencyManagement>
 
+  <dependencies>
```

Comment:
slf4j-simple dependency is _added_ here, but _removed_ from the `jetty-start-mojo-multi-module-single-war-it` pom.xml - seems counterintuitive?

Reply:
The `slf4j-simple` isn't used by the jetty-maven-plugin from the war, it's used from the project.
There's no output during the jetty-maven-plugin execution from this project without this entry.